sources/stylesheets/components/misc/_content-item.scss

```diff
@@ -312,6 +362,24 @@
   box-sizing: border-box;
 }
 
+.product-price {
+  font-weight: 700;
+  font-size: 24px;
```

Comment:
These values no longer seem necessary because they are now in the hands of the design editor.